internal/service/project/data_source_project_test.go

```diff
@@ -15,26 +14,23 @@ import (
 )
 
 func TestAccProjectDSProject_byID(t *testing.T) {
-	projectName := acctest.RandomWithPrefix("test-acc")
-	orgID := os.Getenv("MONGODB_ATLAS_ORG_ID")
-	teamsIds := strings.Split(os.Getenv("MONGODB_ATLAS_TEAMS_IDS"), ",")
-	if len(teamsIds) < 2 {
-		t.Skip("`MONGODB_ATLAS_TEAMS_IDS` must have 2 team ids for this acceptance testing")
-	}
-
+	var (
+		projectName = acctest.RandomWithPrefix("test-acc")
+		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
+	)
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:                 func() { acc.PreCheckBasic(t); acc.PreCheckTeamsIds(t) },
+		PreCheck:                 func() { acc.PreCheckBasic(t); acc.PreCheckProjectTeamsIds(t, 2) },
```

Comment:
```suggestion
		PreCheck:                 func() { acc.PreCheckBasic(t); acc.PreCheckProjectTeamsIdsWithMinSize(t, 2) },
```

nit: for clarity I would suggest a more specific method name

Reply:
I can do that but I normally like to say "with..." when there are multiple related methods, there is only one method in this case

Reply:
renamed to: func PreCheckProjectTeamsIdsWithMinCount(tb testing.TB, minTeamsCount int)